allure-report-face/src/variables.css

```diff
@@ -1,10 +1,10 @@
 $brand-blue: #0044bb;
 $brand-red: #ff0000;
 
-$color-passed: #3bbd25;
-$color-failed: #ff0000;
-$color-broken: #ffa924;
-$color-canceled: #999999;
+$color-passed: #97cc64;
+$color-failed: #fd5a3e;
+$color-broken: #ffd963;
```

Comment:
all of them should be in one group (of brightness, light etc...) or they will be something from different pallettes